src/main/java/net/smartcosmos/edge/things/domain/RestEdgeThingResponseDto.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.edge.things.domain;
+
+import java.beans.ConstructorProperties;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties({ "version" })
+@ApiModel(description = "Unpaged response received when querying for Things.")
+public class RestEdgeThingResponseDto<T> {
+
+    private final List<T> data;
```

Comment:
Looks like the model is missing the `notFound` part that is defined for the core service:
```
{
    "notFound": [
        "urn:thing:uuid:62a8d3a2-6aca-49b9-825a-147a8ee3773d"
    ]
}
```

Thus, edge-things and things core don't have a consistent API right now.